rq/job.py

```diff
@@ -494,8 +492,9 @@ def to_dict(self, include_meta=True):
             obj['status'] = self._status
         if self._dependency_id is not None:
             obj['dependency_id'] = self._dependency_id
-        if self._dependency_id is not None:
-            obj['dependents_ids'] = self._dependents_ids
+        _dependents_ids = list(self.dependent_ids)
```

Comment:
Conversion of `dependent_ids` to list should be moved to `def dependent_ids`. The docstring on `dependent_ids` also mentions that it returns `a list of ids`.